src/app.js

```diff
@@ -6,7 +6,7 @@ import accepts from 'accepts'
 import redis from 'redis'
 
 import AppConstants from './app-constants.js'
-import { initFluentBundles, updateAppLocale } from './utils/fluent.js'
+import { initFluentBundles, updateAppLocale, loadBreachesIntoApp } from './utils/index.js'
```

Comment:
I think this answers my comment below re `/utils/index.js`... 

I miss the context I get from reading the actual module where each function comes from.  Also by linking directly, I can find the related file faster – using the index with wildcard loses the paper-trail for me, or has me open the index file only to find a dead-end? 😬

Reply:
The thing I like about having consolidated index imports is that we don't end up with extra lines per each file.
In terms of tracking files, more IDEs (VS code) usually has a way to get there easily, for example, by doing `cmd+click` while hovering over the function you are interested in tracing.

Does that work?

Reply:
Great tip about `cmd+click`!  In this case I had a hard time finding the file/function in GitHub.  I don't want to mess with your flow – I'm sure it's fine.  

Final thought though – doesn't the wildcard index you're using auto-export everything?  Even functions that are not used in the codebase?  I'm concerned about the eventual cruft that will get loaded even though it's not used.  In particular for front-end this would be unwanted behavior for tree-shaking, etc?

Reply:
I'm not sure I'm interpreting this correctly, but wouldn't that depend on what you import instead of export? 
If you are concerned about private functions being exported by mistake, the exports are bounded in the individual files and the wildcard in `index.js` cannot export more than what's allowed by the underlying files.